src/Components/ProfileListPage.js

```diff
@@ -161,19 +162,7 @@ export default function ProfileListPage() {
             {getSubtitleText(typeName, items)}
           </Typography>
         </Box>
-        {canEdit && canDelete && (
-          <Tooltip title="Delete list">
-            <IconButton
-              size="large"
-              variant="contained"
-              color="inherit"
-              sx={{ marginLeft: "0.75rem" }}
-              onClick={onDelete}
-            >
-              <X />
-            </IconButton>
-          </Tooltip>
-        )}
+        {canEdit && canDelete && <ProfileListDropMenu onDelete={onDelete} />}
```

Comment:
In a follow-up, it might be cool to provide this DropMenu for users who aren't the owner as well, with limited features (maybe just Copy Link, later we could add reporting features if we see there is watchlist abuse somehow, etc...).

Before doing this, though, I might want to refactor the variables I made above, for clarity.  For example, it is kinda confusing that `canEdit` indicates that the user has permission to edit this playlist, while `canDelete` doesn't actually indicate that the user has permission to delete it, but rather that it is the type of list that can be deleted.  This is a side-effect of `canDelete` being created before you could view other users' lists, but now I think this can be cleaned up.

Reply:
Good call - will revise in upcoming commit.